docs/support_files/api_reference.md

```diff
@@ -125,8 +125,8 @@ setDefinitionFunctionWrapper(function(fn, options) {
         // call a method on world
         this.doScreenshot();
         // rethrow error to avoid swallowing failure
-        throw error;
-      })  
+        throw error
```

Comment:
Can we add semicolons to this example where appropriate. I think i prefer having them (and will be updating cucumber-js source to have them soon)

Reply:
Sure, I also prefer semicolons - will add them if everybody agrees